default/generated/azure/tests/data/03-azure-database-config/application-prod.properties

```diff
@@ -0,0 +1,3 @@
+spring.data.mongodb.uri=mongodb://mongouser:deepsecret@mongoserver.contoso.com:27017/mydatabase
+spring.data.mongodb.username=dbuser
+spring.data.mongodb.password=dbpass
```

Comment:
use placeholder, it might be scanned out by some security rule

Reply:
sure, replaced all existing passwords to placeholders